extension/oidcextension/authenticator.go

```diff
@@ -144,6 +150,20 @@ func (o *oidcAuthenticator) StreamInterceptor(srv interface{}, str grpc.ServerSt
 	return o.streamInterceptor(srv, str, info, handler, o.Authenticate)
 }
 
+// ToServerOptions builds a set of server options ready to be used by the gRPC server
+func (o *oidcAuthenticator) ToServerOptions() ([]grpc.ServerOption, error) {
+	// perhaps we should use a timeout here?
+	// TODO: we need a hook to call auth.Close()
+	if err := o.Start(context.Background(), o.host); err != nil {
```

Comment:
Isn't `Start` called by the collector? Why do you need to call it explicitly here?

Reply:
Oh thats a good point, I overlooked that in the change to an extension